lib/streamlit/delta_generator.py

```diff
@@ -320,6 +320,10 @@ def container(self):
         msg.metadata.parent_block.path[:] = self._cursor.path
         msg.metadata.delta_id = self._cursor.index
 
+        msg.metadata.container.collapsible = collapsible or False
+        if label:
+            msg.metadata.container.label = label
```

Comment:
I was on the fence about adding a test for this. Let me know if you think we need.